server/views/pages/prisonerList/index.njk

```diff
@@ -21,7 +21,7 @@
     <div class="govuk-grid-column-full">
       <h1 class="govuk-heading-l">Manage learning and work progress</h1>
 
-      {% include './partials/searchBox.njk' %}
+      {# {% include './partials/searchBox.njk' %} #}
```

Comment:
Commented this out, whilst it exists in the UI, it doesn't have any functionality.